core/templates/pages/topic-viewer-page/navbar-breadcrumb/topic-viewer-navbar-breadcrumb.component.html

```diff
@@ -1,9 +1,15 @@
-<div dir="auto" class="nav navbar-nav oppia-navbar-breadcrumb">
+<div *ngIf="!isLanguageRTL()" dir="auto" class="nav navbar-nav oppia-navbar-breadcrumb">
```

Comment:
Do you think there needs to be a TODO/issue cataloging the cases where ngIf is used in templates like this, so that we can generalize them in the future? (Or is this the "final state"?)

If the former, please file an issue for this and link to it in this comment thread. Thanks!

Reply:
As far as I can tell, this is the only place where this is used other than the navbar. I think an issue to remove this would be fine since the same can be achieved using flexbox, I believe.